data.sql

```diff
@@ -84,4 +84,43 @@ UPDATE animals
 SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
 WHERE name = 'Boarmon';
 
-SELECT * FROM animals;
\ No newline at end of file
+
+--  new tables data to link up vets, visits and spcialization
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('William Tatcher', 45, '2000-04-23');
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('Maisy Smith', 26, '2019-01-17');
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES 
+    ('Stephanie Mendez', 64, '1981-05-04'),
+    ('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specializations (vet_id, species_id)
+VALUES
+    (1, 1),
+    (3, 2),
+    (2, 1),
+    (3, 1);
+    
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES
+    (2, 3, '2020-05-24'),
+    (2, 2, '2020-07-22'),
+    (1, 4, '2021-02-02'),
+    (3, 1, '2020-01-05'),
+    (3, 1, '2020-03-08'),
+    (3, 1, '2020-05-14'),
+    (4, 2, '2021-05-04'),
+    (18, 4, '2021-02-24'),
+    (17, 1, '2019-12-21'),
+    (17, 2, '2020-08-10'),
+    (18, 4, '2021-04-07'),
+    (19, 3, '2019-09-29'),
+    (20, 2, '2020-10-03'),
+    (20, 2, '2020-11-04'),
+    (4, 1, '2019-01-24'),
+    (4, 1, '2019-05-15'),
+    (4, 1, '2020-02-27'),
+    (4, 1, '2020-08-03'),
+    (22, 3, '2020-05-24'),
+    (22, 1, '2021-01-11');
```

Comment:
It seems you have entered the wrong data in your `data.sql` file.

Kindly check to ensure the correctness of your query results 🛠 🙏